cli/cmd/root.go

```diff
@@ -285,8 +290,10 @@ func (options *proxyConfigOptions) flagSet(e pflag.ErrorHandling) *pflag.FlagSet
 	// Deprecated flags
 	flags.StringVar(&options.proxyMemoryRequest, "proxy-memory", options.proxyMemoryRequest, "Amount of Memory that the proxy sidecar requests")
 	flags.StringVar(&options.proxyCPURequest, "proxy-cpu", options.proxyCPURequest, "Amount of CPU units that the proxy sidecar requests")
+	flags.StringVarP(&options.linkerdVersion, "linkerd-version", "", options.linkerdVersion, "Tag to be used for Linkerd images")
```

Comment:
related to previous comment, remove?

Reply:
I think it's valuable to leave this, especially for development. but i don't think it should be visible by default

Reply:
i don't think it's used anymore?
```bash
$ bin/go-run cli install --linkerd-version edge-1.2.3 | grep edge-1.2.3
Flag --linkerd-version has been deprecated, use --proxy-version instead
```

Reply:
Ah, I think that's wrong then. We still need a way to override the linkerd version. If we want to call it `control-plane-version` or something, that's fine with me.

Reply:
Yeah, I did rename it to `control-plane-version` prior to submitting this PR, then back out of it, because of the changes I had to make to get it into the [proxy-init image tag](https://github.com/linkerd/linkerd2/blob/8323e104fbf6921e3c9b038acd96be3f33b5a860/pkg/inject/inject.go#L643).

Before I go ahead and include this change, can we not just use something like `kubectl edit` or `kubectl set image` to change the control plane version, for development purposes?

Reply:
Actually, if we're gonna include the control plane version in the global config, then this won't be a problem. I can fix this in my next commit.